dataprofiler/profilers/profile_builder.py

```diff
@@ -1091,6 +1097,7 @@ class StructuredProfiler(BaseProfiler):
 
     _default_labeler_type = 'structured'
     _option_class = StructuredOptions
+    _allowed_external_data_types = (list, pd.Series, pd.DataFrame)
```

Comment:
Should we allow list? We haven't in the past and I don't think that's really necessary.